src/components/MultiHopTrade/components/TradeConfirm/ReceiveSummary.tsx

```diff
@@ -78,6 +80,10 @@ export const ReceiveSummary: FC<ReceiveSummaryProps> = memo(
     const greenColor = useColorModeValue('green.600', 'green.200')
     const textColor = useColorModeValue('gray.800', 'whiteAlpha.900')
 
+    // use the fee data from the actual quote in case it varies from the theoretical calculation
+    const affiliateBps = useAppSelector(selectActiveQuoteAffiliateBps)
```

Comment:
Won't this rug the "free trades" flow?

Reply:
solved in 4c28a00c7941ac2e2c4d2320b442c6a89ee3d7fa